src/lib/pages/start-workflow.svelte

```diff
@@ -216,6 +216,7 @@
         bind:value={workflowId}
         label="Workflow ID"
         class="w-full grow"
+        maxLength={36}
```

Comment:
Just noting that whatever we do here we'll likely also need to do for the Workflow ID input in the schedule form.

Reply:
Actually, looking at the original issue here and I think it lies with the custom search attribute input not with the inputs on these forms.

Reply:
^ Yup this is for the Int type input for a Search Attribute

Reply:
Thank you both!